src/v3/src/transformer/oktaVerify/transformOktaVerifyEnrollPoll.ts

```diff
@@ -127,7 +168,149 @@ export const transformOktaVerifyEnrollPoll: IdxStepTransformer = ({ transaction,
+            ),
+          },
+        } as DescriptionElement,
+        {
+          type: 'Description',
+          noMargin: true,
+          options: {
+            content: loc('oie.enroll.okta_verify.setup.customUri.makeSureHaveOV', 'login'),
+          },
+        } as DescriptionElement,
+      );
+    } else {
+      sameDeviceOVElements.push(
+        {
+          type: 'Description',
+          noMargin: true,
+          options: {
+            contentClassname: 'desktop-instructions ov-info',
+            content: loc(
+              'oie.enroll.okta_verify.setup.customUri.makeSureHaveOVToContinue', 'login',
+            ),
+          },
+        } as DescriptionElement,
+      );
+    }
+    if (deviceMap.platformLC) {
+      let platformSupported = true;
+      let imageSrc = '/img/appstore/';
+      switch (deviceMap.platformLC) {
+        case 'android':
+          imageSrc += 'google-play-store.svg';
+          break;
+
+        case 'ios':
+          imageSrc += 'apple-app-store.svg';
+          break;
+
+        case 'windows':
+          imageSrc += 'windows-app-store.svg';
+          break;
+
+        case 'osx':
+          imageSrc += 'osx-app-store.svg';
+          break;
+
+        default:
+          platformSupported = false;
+      }
+      if (platformSupported) {
+        sameDeviceOVElements.push(
+          {
+            type: 'ImageLink',
+            options: {
+              url: deviceMap.downloadHref,
+              altText: loc('enroll.oda.step3', 'login'),
+              imageSrc,
+              linkClassName: 'app-store-link',
+              imageClassName: 'app-store-logo',
+            },
+          } as ImageLinkElement,
+        );
+      }
+    }
+    sameDeviceOVElements.push(
+      {
+        type: 'Description',
+        noMargin: true,
+        options: {
+          content: loc(
+            'oie.enroll.okta_verify.setup.customUri.setup',
+            'login',
+            [],
+            { $1: { element: 'span', attributes: { class: 'strong' } } },
+          ),
+        },
+      } as DescriptionElement,
+    );
+
+    if (setupOVUrl) {
+      sameDeviceOVElements.push(
+        {
+          type: 'Button',
+          label: loc('oie.enroll.okta_verify.setup.title', 'login'),
+          options: {
+            step: '',
+            type: ButtonType.BUTTON,
+            variant: 'primary',
+            onClick: () => {
+              if (isAndroid()) {
+                Util.redirectWithFormGet(setupOVUrl);
+              } else {
+                window.location.assign(setupOVUrl);
+              }
+            },
+          },
+        } as ButtonElement,
+      );
+    }
+
+    if (showAnotherDeviceLink) {
+      const labelParts = loc(
+        deviceMap.isDesktop ? 'oie.enroll.okta_verify.setup.customUri.orOnMobile'
+          : 'oie.enroll.okta_verify.setup.customUri.orMobile',
+        'login',
+      ).split('<$1>');
```

Comment:
Personally, I am not a fan of doing this, while it may seem ok for this string, this sets a bad example, strings should not be broken apart like this. What we should really do is create an HTML parser specifically for the Stepper Link to preserve the intent of the string. 

Can you please create a Jira for this work and add a comment above this code with the Jira Number so we can circle back to this when we have more time?

Reply:
added OKTA-701606

Reply:
I ended up looking into this a bit more and it seems we can solve it pretty easily with a small modification, see below: 

```
// inside this file: 
const parserOptions: HTMLReactParserOptions = {};
      parserOptions.replace = getStepperLinkReplacerFn(parserOptions, () => {console.log('test')}, 'setup-ov');
      const ovSetupLink: DescriptionElement = {
        type: 'Description',
        parserOptions,
        options: {
          content: loc(
            deviceMap.isDesktop ? 'oie.enroll.okta_verify.setup.customUri.orOnMobile'
              : 'oie.enroll.okta_verify.setup.customUri.orMobile',
            'login',
            undefined,
            { $1: { element: 'a', attributes: { href: '#', class: 'setup-ov' } } },
          ),
        },
      }
      sameDeviceOVElements.push(ovSetupLink);
```

Then inside `src/v3/src/util/htmlContentParserUtils.tsx` you can add this function: 
```
// ... 
export const getStepperLinkReplacerFn = (
  parserOptions: HTMLReactParserOptions,
  onClick: (() => void),
  dataSe: string,
): HTMLReactParserOptions['replace'] => (
  (node: DOMNode) => {
    const {
      attribs = {},
      children = [],
      name = undefined,
      type = undefined,
    } = (node instanceof Element) ? node : {};
    if (node instanceof Element && type === 'tag' && name === 'a') {
      const props = attributesToProps(attribs);
      return (
        <Link
          // eslint-disable-next-line react/jsx-props-no-spreading
          {...props}
          component="button"
          variant="body1"
          role="link"
          onClick={onClick}
          data-se={dataSe}
          sx={{
            '&:hover': {
              cursor: 'pointer',
              verticalAlign: 'baseline',
            },
          }}
        >
          {domToReact(children, parserOptions)}
        </Link>
      );
    }
    return undefined;
  }
);
// ...
```

The only change needed would be to modify the onClick handler to use the Stepper Context to modify the step index on click (we can help with this).